src/main/java/io/debezium/connector/informix/InformixConnection.java

```diff
@@ -136,4 +140,20 @@ public String quotedColumnIdString(String columnName) {
         // TODO: Unless DELIMIDENT is set, column names cannot be quoted
         return columnName;
     }
+
+    public Table getTableSchemaFromTableId(TableId tableId) throws SQLException {
+        final TableEditor tableEditor = Table.editor().tableId(tableId);
+        final DatabaseMetaData metadata = connection().getMetaData();
+        final ResultSet columns = metadata.getColumns(
+                tableId.catalog(),
+                tableId.schema(),
+                tableId.table(),
+                null);
+        while (columns.next()) {
+            readTableColumn(columns, tableId, null).ifPresent(columnEditor -> tableEditor.addColumns(columnEditor.create()));
+        }
```

Comment:
As indicated in the other PR that this one is based upon, wrap this within try-with-resources.